cmd/main.go

```diff
@@ -6,8 +6,6 @@ import (
 
 	"github.com/spf13/cobra"
 
```

Comment:
Looks like removing the dependency actually removes the functionality of auto-setting the GOMAXPROCS based on K8S requests/limits and not just the logging:
https://github.com/argoproj/argo-cd/pull/16528